### ISSUE TITLE
External repository completions don't work with query sync

### ISSUE THREAD

**Issue by tpasternak** [bazelbuild/intellij#6843]
### Description of the bug:

It's because we don't call dump_repo_mapping in qsync mode.

cc @mtoader just to keep you in the loop, I think I will find some time to fix it next week

**Comment by mtoader** [comment]
Yea :) i did ask if i should look into adding that in the original request.

**Comment by tpasternak** [comment]
Yeah, I thought the query sync was broken either way back then :D
